shell/platform/android/io/flutter/embedding/android/FlutterActivityAndFragmentDelegate.java

```diff
@@ -346,6 +347,10 @@ private void doInitialFlutterViewRun() {
       return;
     }
 
+    if (!FlutterInjector.instance().flutterLoader().initialized()) {
+      throw new AssertionError("Flutter UI can only be started once the Flutter engine's running.");
```

Comment:
If users hit this, will they know what to do? Perhaps we can guide them to add-to-app guide or something?